filter/index.js

```diff
@@ -1,21 +1,20 @@
+var _ = require('lodash')
+
+var numbers = [1,2,3,4,5]
+
+var sum = _.sum(numbers)
+console.log(sum);
+
 function onlyEven(array) {
-  const evens = [];
-  for (let i = 0; i < array.length; i++) {
-    if (array[i] % 2 === 0) {
-      evens.push(array[i]);
-    }
-  }
-  return evens;
+return  array.filter(x => x%2===0)
 }
 
-// refactor with filter!! OMG!
-
 function onlyOneWord(array) {
-  //where are the snacks?
+  return array.filter(x => x.includes(" ")==false)
 }
 
 function positiveRowsOnly(array) {
-  //is Dan human?
+return array.filter(x => x.some(y => y < 0)===false)
```

Comment:
Wow! kudos! putting the false at the end buries the lead, declare a falsy statement initially`!` to give the reader an idea of whats going on in the code.

```js
return array.filter(x => !x.some(y => y < 0))
```